docs/source/neuromodulation.rst

```diff
@@ -0,0 +1,22 @@
+===============
+Neuromodulation
+===============
+
+A neuromodulatory event influences synaptic behaviours by simulating additional neurontransmitters. It is implemented by linking from projection fibers to existing synapses as illustrated below:
+
+.. image:: ./img/neuromodulation.png
```

Comment:
it would be nice to have the picture in as neuromodulation.darwio.svg so it can be easily changed in the future

Reply:
I didn't draw the diagram but captured from scientiest's slides. I tried to convert from png to drawio.svg , but vscode is not able to open it.